src/cbc_sdk/platform/users.py

```diff
@@ -370,10 +380,11 @@ def bulk_delete(cls, users):
     def disable_all_access(self):
         """Disables all access profiles held by ths user."""
         grant = self.grant()
-        for profile in grant.profiles_:
-            profile.set_disabled(True)
-            grant.touch()
-        grant.save()
+        if grant:
```

Comment:
We should probably raise an error if there is no grant to disable so it doesn't appear to succeed when the legacy form is still being used

Reply:
Apply to all other locations you added the grant conditional

Reply:
Except for add_profiles, because there's a reasonable solution there (create a grant that has the profiles in it).

Reply:
Addressed this in most recent commit.  Let me know what you think.